package.json

```diff
@@ -23,6 +23,9 @@
   "bugs": {
     "url": "http://github.com/elastic/kibana/issues"
   },
+  "config": {
```

Comment:
Needed so pptr doesn't download chromium, which is the default behavior.

Reply:
Some of the yarn commands still download chromium (`yarn add` for instance). I'm investigating why that its.

Reply:
Overall, the code changes in this PR look good to me. Give me a ping when you find out more about this.

Reply:
ops has given the sign-off on this since it's only downloaded when `yarn remove` is ran. It's definitely a bug in yarn, which I'll follow up on.